lib/osf-components/addon/components/search-help-modal/component.ts

```diff
@@ -16,11 +12,13 @@ import template from './template';
  * }}
  * ```
  */
-@layout(template, styles)
-export default class SearchHelpModal extends Component {
-    @service router!: RouterService;
 
-    isOpen = false;
+interface InputArgs {
+    isOpen: false;
```

Comment:
I think this shouldn't be an arg, but should be a `@tracked` property of the component class.